src/build.gradle.kts

```diff
@@ -0,0 +1,27 @@
+subprojects {
+    apply<JavaPlugin>()
+
+    dependencies {
+        add("testImplementation", "junit:junit")
```

Comment:
Can we add `add("testImplementation", "org.spockframework:spock-core")` 
I added this because not all the Spock tests were compiling in IntelliJ.
I think you can then remove it from the `src/core/build.gradle.kts`

Reply:
@ham1 , thanks I'll check that. Frankly speaking, I did not verify Spock-based tests yet.

PS. `src/core/build.gradle.kts` does not mention `spock-core`

PPS. `/build.gradle.kts` does mention `spock-core` in order to set the version to be used across all the modules. `/build.gradle.kts`  defines version constraints, and it does not really add `spock` dependency.

Reply:
@ham1 , I've pushed the change to compile and execute Groovy-based tests. I've added `groovy` plugin for Gradle, and moved groovy files to `test/groovy/...` subfolder